testdata/config.toml

```diff
@@ -1,5 +1,5 @@
 Name              = "koding"
-Enabled           = false
+Enabled           = true
```

Comment:
Yeah, I should have left an extra comment about this, but the default test server struct expects this field to be `true`:

https://github.com/koding/multiconfig/blob/master/multiconfig_test.go#L42

This is also consistent with https://github.com/koding/multiconfig/blob/master/testdata/config.json

I'm actually not sure how the TOML tests were passing previously.